### ISSUE TITLE
Stellargraph & ktrain on Python 3.8

### ISSUE THREAD

**Issue by ilos-vigil** [amaiya/ktrain#373]
**TLDR:** Ktrain support Python 3.6-3.8 while _the forked_ stellargraph explicitly require Python >=3.5.0 and <3.8.0

I wanted to generate ktrain documentation with pdoc3 on Python 3.8. When installing stellargraph, i encountered this error

```
ERROR: Package 'stellargraph' requires a different Python: 3.8.7 not in '<3.8.0,>=3.5.0'
```

Ktrain [setup.py](https://github.com/amaiya/ktrain/blob/master/setup.py#L75-L77) mention support Python 3.6-3.8, but _the forked_ stellargraph [setup.py](https://github.com/amaiya/stellargraph/blob/develop/setup.py#L70) explicitly require Python >=3.5.0 and <3.8.0.

**Comment by amaiya** [comment]
Hi @ilos-vigil :  Thanks for pointing this out.  I'll look into fixing this.

**Comment by amaiya** [comment]
@ilos-vigil There is now an [API Docs](https://amaiya.github.io/ktrain/index.html) link at the top of the README created using `pdoc3`.  Thanks for the suggestion to use `pdoc3`.

I have simply updated the  Python version requirement in the `stellargraph` fork.    If you test it and it does not work, please do let me know.  Otherwise, I'll close this issue for now.

**Comment by ilos-vigil** [comment]
Everything work when i tried it (creating documentation with pdoc3) and run example notebook.

However, since the _forked_ stellargraph has 1680 commit, it took a while to clone with slow connection (~200 kbps). Unfortunately pip doesn't support clone with depth 1 ([source](https://github.com/pypa/pip/issues/2432)). I advice to mention alternative way to install _forked_ stellargraph on slow connection.

```
pip install https://github.com/amaiya/stellargraph/archive/refs/heads/no_tf_dep_082.zip
```

**Comment by amaiya** [comment]
Great - I've updated the README with your alternatives for both `stellargraph` and `eli5`.  Thanks!
